### PR TITLE
Typos Chapters 1, 2, 3 & 4

### PR DESCRIPTION
**Pull request by ARPL-c-63h**
### Pull request on main. No solutions

cc @vrom911

### CODE REVIEW

File: src/Chapter1.hs [hunk 1]
```diff
@@ -284,7 +284,7 @@ ghci> 1 + 2
 
 > It may look scary to you, but we will cover all this 'Num' and "=>" later. For
   now, you can think of this as a polymorphic function — in this case, the
-  operator, that can work with any numeric types, including 'Int's, 'Double's,
```
Comment (vrom911) [inline src/Chapter1.hs:287]: This is the haddock syntax to have a link on `Int` documentation. As these are Haskell comment format of info we decided to go with it 🙂

**Review by vrom911 — APPROVED** [review]
Thanks, @ARPL-c-63h! 👍🏼 
Could you please remove the additional file, so we can merge the improvements right away? 🙂

File: src/Chapter1.hs (hunk 1 above)
Comment (ARPL-c-63h) [reply src/Chapter1.hs:287]: Makes sense. Didn't know that. Not much experience with Haddock. Better look into that. : )

**Review by ARPL-c-63h — COMMENTED** [review]
File deleted.

**Review by chshersh — APPROVED** [review]
Thanks! 👍🏻

**Comment by ARPL-c-63h** [comment]
@chshersh: You're welcome. I was a bit hesitant since we got off on the wrong foot (because of [this](https://github.com/kowainik/relude/pull/300)). Glad it worked out this time.

**Comment by chshersh** [comment]
@ARPL-c-63h No worries, no hard feelings left! I appreciate you contributing improvements to our projects 🙂

**Comment by ARPL-c-63h** [comment]
@chshersh And you, more than anyone else, know that it is almost impossible to resist the cuteness that is @vrom911 ; )
And now I've probably overstepped a whole different set of boundaries! Ah well : )
Until next time!

**Pull request merged**